### DISCUSSION TITLE
How to fix recording issues when using PredictPose DLC?

### DISCUSSION THREAD

**Discussion by MilanNedoma** [bonsai-rx/bonsai#1733] [Q&A]
Good day!

I am trying to use the PredictPose node from the DLC package and I have been encountering some serious problems regarding the video output. Below I am attaching a screenshot for reference. After checking all of the parameter files to ensure that fps is set to 30, as well as within Bonsai, a 1 minute long video comes out as 25 second long and playing as if it was shot using a timelapse. Lowering the fps makes the video longer, which should not even work.
This problem only occurs when using the DLC package, specifically the PredictPose node, as without it, everything runs smoothly.

![image](https://github.com/bonsai-rx/bonsai/assets/140824061/3b6ec187-2732-46ff-bf00-b7fa57dfd368)
[GrimaceTesting.zip](https://github.com/bonsai-rx/bonsai/files/14946582/GrimaceTesting.zip)

Using:
 CUDA 11.2
 cuDNN 8.1.0
 tensorflow 2.10

Any ideas or suggestions?

Thank you very much in advance!

**Comment by bruno-f-cruz** [comment]
Does this answer your question? https://github.com/orgs/bonsai-rx/discussions/1078
